RELEASE.md

```diff
@@ -0,0 +1,38 @@
+# Release
+
+* Update [CHANGELOG.md](https://github.com/lektor/lektor-atom/blob/master/CHANGELOG.md)
+
+* Update package version on [setup.py](https://github.com/lektor/lektor-atom/blob/master/setup.py)
+
+* Commit changes
+  ```bash
+  git add .
+  git commit -m "Prepare release version"
+  ```
+
+* Tag the release
+  ```bash
+  git tag -a vX.X.X -m 'Release version'
```

Comment:
Shouldn't the message be something like `Version X.X.X` or `Lektor-atom X.X.X`?

Reply:
Yep, agreed